### PR TITLE
Switch ExecutionStatus from Pending to Unknown

### PR DESCRIPTION
**Pull request by evgenykuzyakov**
Return `Unknown` status instead of `Pending` for ExecutionOutcome.
Return `NotStarted` for final status, when the first status is `Unknown`.
Change nearlib CI and near-shell dependency from staging to master.

### CODE REVIEW

**Comment by evgenykuzyakov** [comment]
Depends on https://github.com/nearprotocol/nearlib/pull/81

File: chain/client/src/view_client.rs [hunk 1]
```diff
@@ -71,12 +71,14 @@ impl ViewClientActor {
...
         let mut looking_for_id = (*hash).into();
+        let num_outcomes = outcomes.len();
         let status = outcomes
             .iter()
             .find_map(|outcome_with_id| {
                 if outcome_with_id.id == looking_for_id {
                     match &outcome_with_id.outcome.status {
-                        ExecutionStatusView::Pending => Some(FinalExecutionStatus::Started),
+                        ExecutionStatusView::Unknown if num_outcomes == 1 => Some(FinalExecutionStatus::NotStarted),
+                        ExecutionStatusView::Unknown => Some(FinalExecutionStatus::Started),
```
Comment (ailisp) [inline chain/client/src/view_client.rs]: Great we'll have an idea whether it's started or not

File: chain/client/src/view_client.rs [hunk 2]
```diff
@@ -43,7 +43,7 @@ impl ViewClientActor {
             Ok(result) => Ok(result.clone().into()),
             Err(err) => match err.kind() {
                 ErrorKind::DBNotFoundErr(_) => {
-                    Ok(ExecutionOutcome { status: ExecutionStatus::Pending, ..Default::default() }
+                    Ok(ExecutionOutcome { status: ExecutionStatus::Unknown, ..Default::default() }
```
Comment (ailisp) [inline chain/client/src/view_client.rs]: Q: is this the only code path lead to `Unknown`? What does DBNotFoundErr means?

Comment (evgenykuzyakov) [reply chain/client/src/view_client.rs]: It's not found in the DB, which is likely means the transaction/receipt was not yet processed or doesn't exist at all, e.g. fake hash

**Pull request merged**